File: BAEKJOON/Python/8_math/Prob_10250.py
T = 0
H = 0
W = 0
N = 0
row = 0
column = 0
T = int(input())

for i in range (0,T,1):
    H, W, N = map(int,input().split())

    row = int(N/H + 1)
    column = int(N%H)
    column = column * 100
    if (column == 0):
        column = H * 100
        row = int(N/H)
    
    print(column+row)

File: BAEKJOON/C/8_math/Prob_10250.c
#include <stdio.h>

int main()
{
    int T = 0;
    int H, W, N = 0;
    int row, column = 0;
    scanf("%d", &T);
    for (int i = 0; i<T; i++)
    {
        scanf("%d%d%d",&H, &W, &N);

        row = N/H + 1;
        column = N%H;
        column = column * 100;
        if (column == 0)
        {
            column = H * 100;
            row = N/H;
        }
        printf("%d\n",(column+row));
    }
    
}